### ISSUE TITLE
Adding AWGN to a signal

### ISSUE THREAD

**Issue by adeeb10abbas** [veeresht/CommPy#94]
Adding Gaussian noise to a signal of a specific SNR. The signal may or may not have noise already

**Comment by BastienTr** [comment]
Hi @adeeb10abbas,

Could you be more precise on your issue. Is this a question? Do you report a bug? Do you ask for a feature?

**Comment by adeeb10abbas** [comment]
Yes sorry about that. I am asking for a feature! Thanks!

**Comment by BastienTr** [comment]
Did you have a look to the

**Comment by adeeb10abbas** [comment]
Yes, this works but partially. I am looking for something that can add noise to a signal that already has some amount of noise.

**Comment by BastienTr** [comment]
Replace `noiseless_signal` with your already noisy signal and it's done, isn't it?

**Comment by adeeb10abbas** [comment]
So I wanted to figure out a way to add less say x amount of noise to a signal that already has y amount of noise (x and y are the snr). The output would be something with a z snr as it can't be a direct x + y as they are ratios. Does that make sense?
